### ISSUE TITLE
Fullscreen window is too top-level

### ISSUE THREAD

**Issue by fkuehne** [pdherbemont/Glasses#36]
With VLC, you can see and use other application's windows while being in fullscreen mode as these are simply drawn above the video output once the other application is activated. This doesn't work here, as the video output is always shown above the other windows.
Just here so I don't forget about it. Easy to fix.

**Comment by CarterA** [comment]
Why would you want to change this? The old behavior isn't standard. Fullscreen is supposed to stay on top of everything else.

**Comment by pdherbemont** [comment]
I tend to agree with Carter here. However, feel free to add a defaults for that. I am sure we'll hear some complaints like this, so if this is easy... "defaults write org.videolan..." is enough to make people happy I guess.

**Comment by fkuehne** [comment]
I'll add an option for that as personally, I tend to view stuff in fullscreen, switch to Adium to talk to friends and switch back to VLC without leaving fullscreen.
The default behavior should probably be to leave fullscreen automatically when Lunettes resigns from being the front most application.

**Comment by pdherbemont** [comment]
Yes.

**Comment by pdherbemont** [comment]
Actually I think it's useful to have it fullscreen and not frontmost, if we have a second screen. Felix?

**Comment by SnidelyW** [comment]
To add another perspective to the above comments, regardless of which video player I am using, my personal preference is to NOT automatically have a player start a video in full-screen mode; the reason being, obviously, that some videos are of such poor quality -- and/or of low resolution -- that they look horrible when blown up to full-screen. I think it would be great to include a user option in the prefs for those users who want to automatically start their videos in full-screen mode, but I would NOT hard code automatic full-screen mode as the default setting.

**Comment by pdherbemont** [comment]
This is a separate issue, and this should not be the default. Please open a separate issue if that's needed.
